### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by carsantello**
Hadn't had approved, due to some mistakes, please enhance the reviewing process for the future!

### CODE REVIEW

File: blogpost.html [hunk 1]
```diff
@@ -15,14 +15,20 @@ <h1> Este es el titulo atractivo e
 				interesante del post</h1>
 				<p>Y este es el parrafo de inicio donde 
 				vamos a explicar las cosas increibles</p>
-				<p> Aqui estamos verificando el apartado para xxx realizar un merge</p>
+				<p> Aqui estamos verificando el apartado para realizar un merge</p>
```
Comment (carsantelo) [inline blogpost.html:18]: got it, the change were cosmetic, nor functional

Comment (carsantelo) [reply blogpost.html:18]: Iḿ ok by proceeding with the changes, go ahead!

**Review by carsantelo — CHANGES_REQUESTED** [review]
please verify & fix the cosmetic issues

**Review by carsantelo — APPROVED** [review]
Great, everything is good so far!

**Comment by carsantello** [comment]
left a merge by branching the emerge branch called fix-typo

**Pull request merged**